styles/global.css

```diff
@@ -1,12 +1,29 @@
-body {
-  margin: 0;
-  background: linear-gradient(
+:root {
+  --background-color: linear-gradient(
     to right,
     rgb(3, 5, 58),
     rgb(3, 1, 29),
     rgb(17, 3, 56)
   );
+  --purple-bg: #534fc6;
+  --purple: #534fc6;
+  --purple-hover: #5955c6;
+  --shadows: 0 1px 2px rgb(0, 0, 0);
+}
+
+[data-theme="light"] {
```

Comment:
Het is nogal raar om ook de kleuren voor het light theme purple enzo te noemen. Misschien een meer generieke naam gebruiken die verwijst naar het object waarvoor het gebruikt wordt of gewoon hoofdkleur of nevenkleur.

Reply:
Done! Thanks om op te merken.